tests/pilet.test.js

```diff
@@ -39,4 +41,43 @@ describe("pilet", () => {
         );
         expect(info.stderr).toBe("");
     });
+
+    it("pilet publish", async (done) => {
+        const pathToBuildDir = path.resolve(process.cwd(), "pilet");
+        const port = 9000;
+        const testFeedUrl = `http://localhost:${port}/api/v1/pilet`;
+        const testFeedKey = "df133a512569cbc85f69788d1b7ff5a909f6bcfe1c9a2794283a2fc35175882c";
+
+        const localFeedService = spawn(`npm run sample-feed`, {
+            cwd: process.cwd(),
+            shell: true
+        });
+
+        await sleep(5000); //wait for feed service to come up
+
+        const publish = new Promise((res, rej) => {
+            exec(`npx pilet publish --fresh --url ${testFeedUrl} --api-key ${testFeedKey}`,
+            {
+                cwd: pathToBuildDir
+            }, (error, stdout, stderr) => {
+                if(error)
+                    rej({error, stdout, stderr})    
+                else
+                    res({stdout, stderr})
+            });
+        })
+         
+        let publishOutput;
+
+        await publish.then(success => {
+            publishOutput = success.stdout.toString();
+        }).catch(err => {
+            publishOutput = err.stdout.toString();
+        });
+        
+        expect(publishOutput).not.toMatch(/\[[0-9]+\]\s+Failed/gm);
+
```

Comment:
OK can we check that the pilet is also really in the feed service, i.e., that `/api/v1/pilet` has 1 pilet? It should have had 0 before publishing.

Reply:
added second `expect` statement that checks if the returned amount of items is 1